build/dockerfiles/Dockerfile

```diff
@@ -45,8 +43,10 @@ RUN \
     npm i -g jackson-converter@1.0.10 && \
     pip3 install dicttoxml xmindparser
 
-RUN wget  https://terminus-dice.oss-cn-hangzhou.aliyuncs.com/installer/nginx_1.20.2-1~bullseye_${ARCH}.deb && \
-        dpkg -i nginx_1.20.2-1~bullseye_${ARCH}.deb
+RUN arch="$(dpkg --print-architecture)"; arch="${arch##*-}"; \
```

Comment:
You just made some changes here, so let's make a judgment: only install niginx for hepa or all-in-one.